bazel/README.md

```diff
@@ -1,10 +1,9 @@
 # Bazel
 
-## Deprecation Notice
+## Altnernative Official Image
 
-This builder is deprecated and will be deleted in an upcoming release.
-The recent version of the image will remain available for use. As a
-replacement, please use one of the supported images at
+A supported `bazel` image, including multiple tagged versions,
```

Comment:
WDYT about replacing 'A supported' with 'The official'?

Reply:
Hmmm, IDK if it's "the" only one.